video_emotion/facial_recognition/facial_recognition.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+import cv2
+import os
+
+dirname = os.path.dirname(__file__)
+CONFIDENCE_MINIMUM = 0.7
+
+opencv_prototxt = os.path.join(
+        dirname, "deploy.prototxt.txt")
+
+opencv_model = os.path.join(
+        dirname, "res10_300x300_ssd_iter_140000_fp16.caffemodel")
+
+# Load model from disk
+net = cv2.dnn.readNetFromCaffe(opencv_prototxt, opencv_model)
+
+def analyze_video(video_path, time_range=None):
+    if time_range is not None:
+        fro, to = time_range
+    cap = cv2.VideoCapture(video_path)
+    dict_of_faces = {}
+    while(cap.isOpened()):
+        # Get frame from video
+        ret, frame = cap.read()
```

Comment:
What the fuck do you need ret for? Isn't it only used in the if-check, where you check if you have received a frame? I might be totally missing the point. :D